application/src/main/java/org/togetherjava/tjbot/commands/free/FreeHelpChannelsRoutine.java

```diff
@@ -0,0 +1,65 @@
+package org.togetherjava.tjbot.commands.free;
+
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.entities.TextChannel;
+import net.dv8tion.jda.api.utils.TimeUtil;
+import org.jetbrains.annotations.NotNull;
+import org.togetherjava.tjbot.commands.Routine;
+
+import java.time.OffsetDateTime;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Routine that automatically marks inactive help channels as free.
+ * <p>
+ * The routine runs every {@value CHECK_INTERVAL_MINUTES} minutes, and checks for help channels that
+ * have been inactive for more than {@value INACTIVE_TIME_MINUTES} minutes.
+ * </p>
+ */
+public class FreeHelpChannelsRoutine implements Routine {
+    private static final int INACTIVE_TIME_MINUTES = 120;
```

Comment:
i would rename this to `INACTIVE_AFTER_MINUTES`. Or even better, dont use `int` but `Duration`: `Duration INACTIVE_AFTER = Duration.ofHours(2)`

Reply:
how do I show it in the javadoc then?

Reply:
Not, `{@value}` only works with primitives and String's.
Can replace it with a `{@link}`?

Reply:
probably, but that would make it less readable.

Reply:
And the advantage is superior maintainability, instead of having to check a lot of documentation for hardcoded values it just works.

The disadvantage of readability isn't even that big, it's very small